src/scripts.js

```diff
@@ -54,6 +56,12 @@ function updateAllRecipeDisplay() {
       <div class="recipe-tags-container">
         ${tagsHTML.toString()}
       </div>
-    </section>`
-  });
-};
+    `
+
+    // TO DO: Put modal event handler in line 63
+    recipeCard.onclick = () => {
+      console.log(`hi! ${recipe.name}`)
+    }
```

Comment:
create each recipe cards `event listener` and have a place holder console log here until we create the modal event handler. If you pull down this branch and run, you can test and see when you click on a recipe card right now the console will print the recipe name!

Reply:
Awesome, looks great!